nextflow_helpers/labels_tw.config

```diff
@@ -1,3 +1,22 @@
+def exitStrat(exitStatus, attempt) {
+  // if there is no exit status, it's likely the process
+  // was killed because a spot instance was reclaimed
+  if (exitStatus == null && attempt < 3) {
```

Comment:
I don't think you need the `attempt < 3` when `maxRetries` is set.

Reply:
unfortunately not, but I realise the function should be restructure to reflect this

Reply:
Ah, I see. I guess you could pass `maxRetries` and use that instead of hard coding 3 but this is fine.

Reply:
Good point! Though I'm not sure how easy it is to access the maxRetries at this stage